osaka/storage/s3.py

```diff
@@ -173,6 +184,22 @@ def reload_obj(self, obj):
         """
         obj.load()
 
+    @staticmethod
+    def _check_credentials():
```

Comment:
i couldn't figure out a way to get this logic into a single line, therefore i made this into a method which returns a boolean

but if we are able to condense this into a single-line it wont be as readable